ivy/backend_handler.py

```diff
@@ -46,9 +48,16 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 _backend_reverse_dict["ivy.functional.backends.torch"] = "torch"
 
 
+_static_backend_dict = dict()
 # Backend Getting/Setting #
 # ----------------------- #
 
```

Comment:
Added this function to basically pre-load all ivy backends. This is later required in the `dynamic_backend` setter functions which basically use `determine_backend_from_args` to determine the backend of the input.

Reply:
Hey @YushaArif99, I think this would add `jax`, `tensorflow` and `torch` as dependencies to the backend handler and in turn to the project, whereas users should be able to instead just install the libraries they wish to work with. I'd suggest looking into ways to check if a library is installed and only pre-loading the backends whose framework is installed in this function

Reply:
Yeah, this logic is not needed anymore actually. I added it because there were some import issues I was facing when using `_determine_backend_from_args`. But they don't seem to be showing up now so I have removed this static_backend logic completely.